src/api/server.ts

```diff
@@ -0,0 +1,74 @@
+import fastify, {HTTPMethods} from 'fastify';
+import logger from '../core/log';
+import fastifyWs from 'fastify-websocket';
+import {transformErrorResp} from '../errors';
+import identityMdw from './middleware/identity';
+const ws_routes = require('./ws_routes');
+const api_routes = require('./api_routes');
```

Comment:
Ok, since we already have started refactoring API server, let's also get rid of the `api_routes` and `ws_routes` files. First of all the handlers themselves should be functions, not classes. It makes no sense to instantiate ones just to simply handle a requests. Given that we don't need to have the mapping between urls and class methods, we can simply attach the handlers here the same way we do in proxy.

Reply:
This should be quite a large change, so, as I proposed earlier, let's leave it for the next PR